boards/ti/lp_mspm0g3507/lp_mspm0g3507.yaml

```diff
@@ -0,0 +1,15 @@
+identifier: lp_mspm0g3507
```

Comment:
```suggestion
identifier: lp_mspm0g3507/mspm0g3507
```

Reply:
I don't fully understand this change for the identifier. Why do I want to do this?

Reply:
Do you mean one specific soc as in `mspm0g3507` or do you mean one specific soc family as in `mspm0g350x` where x can be multiple things? The product page for this board isn't great because it lists a whole bunch of devices and doesn't specify if they come on the board of if they're just compatible. If this only supports `mspm0g3507` then this request can be ignored and marked as resolved, if it supports multiple then it must be added